react/src/README.md

```diff
@@ -0,0 +1,124 @@
+# Mayflower-React
```

Comment:
Did you mean to copy this file into `src/`?  This was living at the package root, and I can't see a reason to duplicate it unless Storybook requires it for some reason.

Reply:
initially `npm start` was barfing saying that it was referencing a file outside of the `src` directory. That is not the case anymore, so I removed it and am pointing to the root one.